docs/source/admin/traffic_ops/using.rst

```diff
@@ -409,23 +412,26 @@ The fields in the Delivery Service view are:
+| Long Description 1                           | Free text field not currently used in configuration. For example, you can use this field to describe your customer type. To be consumed from the APIs by downstream tools such as the Traffic Portal                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 +----------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Long Description 2                           | Free text field not currently used in configuration. To be consumed from the APIs by downstream tools (Portal)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Long Description 2                           | Free text field not currently used in configuration. To be consumed from the APIs by downstream tools such as the Traffic Portal                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
can you get rid of the "such as the Traffic Portal"? Traffic Portal doesn't really use this field, instead it allows you to set this field. That is it.